webpack.config.js

```diff
@@ -19,6 +20,12 @@ const redashBackend = process.env.REDASH_BACKEND || "http://localhost:5000";
 const basePath = path.join(__dirname, "client");
 const appPath = path.join(__dirname, "client", "app");
 
+const tsConfigPath = path.join(basePath, "tsconfig.json");
+
+if (!fs.existsSync(tsConfigPath)) {
+  throw new Error(`Can not find tsconfig: ${tsConfigPath}`);
```

Comment:
I added this line because the initial error I got for the missing tsConfig file in the docker container was missleading (see https://github.com/TypeStrong/ts-loader/issues/405#issuecomment-330108362)

I then figured out that I either need to move tsconfig into `/client` or copy it over in the Dockerfile.